build.cake

```diff
@@ -51,12 +53,11 @@ Task("GetSource")
     .IsDependentOn("Clean")
     .Does(() =>
     {
-	    FilePath sourceZip = DownloadFile("https://codeload.github.com/reactiveui/ReactiveUI/zip/main");
-        Unzip(sourceZip, dependenciesDir);
-        
-        // Need to rename the container directory in the zip file to something consistent
-        var containerDir = GetDirectories(dependenciesDir.Path.FullPath + "/*").First(x => x.GetDirectoryName().StartsWith("ReactiveUI"));
-        MoveDirectory(containerDir, sourceDir);
+        GetSource("Akavache", "master");
+        GetSource("DynamicData", "master");
+        GetSource("ReactiveUI", "main");
+        GetSource("Sextant", "master");
```

Comment:
Let's just change all repositories to use "main" now and then remove this parameter.